README.md

```diff
@@ -101,61 +108,57 @@ class Example extends React.Component {
-| dataURLKey       | string   |                                           | "dataURL" | Field name that base64 of selected image is assign to                            |
-| multiple         | boolean  |                                           | false     | Set true for multiple choose                                                     |
-| maxNumber        | number   |                                           | 1000      | Number of images user can select if mode = "multiple"                            |
-| onError          | function | \(errors, files\) => void                 |           | Called if have error on validate each update                                     |
-| acceptType       | array    | \['jpg', 'gif', 'png'\]                   | \[\]      | Supported image extension                                                        |
-| maxFileSize      | number   |                                           |           | Max image size\(Byte\) \(will use in the image validation\)                      |
-| resolutionType   | string   | "absolute" \| "less" \| "more" \| "ratio" |           | Using for validate image with your width \\\- height resolution that you provide |
-| resolutionWidth  | number   | > 0                                       |           |                                                                                  |
-| resolutionHeight | number   | > 0                                       |           |                                                                                  |
+| parameter | type | options | default | description |
+| :---------------- | :-------- | :----------------------------------------- | :--------- | :-------------------------------------------------------------------------------- |
+| value | array | | [] | List of images |
+| onChange | function | (imageList, addUpdateIndex) => void | | Called when add, update or delete action is called |
+| dataURLKey | string | | dataURL | Customized field name that base64 of selected image is assigned to |
+| multiple | boolean | | false | Set `true` for multiple chooses |
+| maxNumber | number | | 1000 | Number of images user can select if mode = `multiple` |
+| onError | function | (errors, files) => void | | Called when it has errors |
+| acceptType | array | ['jpg', 'gif', 'png'] | [] | The file extension(s) to upload |
+| maxFileSize | number | | | Max image size (Byte) and it is used in validation |
+| resolutionType | string | 'absolute' \| 'less' \| 'more' \| 'ratio' | | Using for image validation with provided width & height |
+| resolutionWidth | number | > 0 | | |
+| resolutionHeight | number | > 0 | | |
+
+### Note
+
+**resolutionType**
+
+| value             | description |
+| :---------------- | :---------- |
+| absolute | resolutionWidth === resolutionHeight |
```

Comment:
the correct is image's width === resolutionWidth && image's height === resolutionHeight